src/main/java/com/w6/controller/EndpointController.java

```diff
@@ -159,7 +166,96 @@ public ModelAndView parse() throws IOException
             return new ModelAndView(W6_VIEW);
         }
     }
+    @RequestMapping(value = "relevant", method = RequestMethod.GET)
+    public ModelAndView relevant() throws IOException
+    {
+        ModelAndView modelAndView = new ModelAndView(QUERY_VIEW);
+        ArrayList<Article> documents;
+        try { 
+            documents = solrClient.getDocuments("   Involved\n" +
+"   Incident\n" +
+"   Staff\n" +
+"   IMC\n" +
+"   aid\n" +
+"   Office\n" +
+"    security\n" +
+"    NGO\n" +
+"    killed\n" +
+"    Afghanistan\n" +
+"    Deaths\n" +
+"    police\n" +
+"    workers\n" +
+"    Afghan\n" +
+"    Taliban\n" +
+"    Vehicle\n" +
+"    international\n" +
+"    government\n" +
+"    humanitarian\n" +
+"    Security\n" +
+"    UN\n" +
+"    Incidents\n" +
+"    Pakistan\n" +
+"    kidnapped\n" +
+"    armed\n" +
+"    hospital\n" +
+"    Injuries\n" +
+"    WFP\n" +
+"    CMT \n" +
+"    HQ\n" +
+"    AOG\n" +
+"    Evacuation\n" +
+"    Relocation\n" +
+"    Weapon\n" +
+"    Violation\n" +
+"    Property\n" +
+"    Equipment\n" +
+"    members\n" +
+"    military\n" +
+"    officials\n" +
+"    Chad\n" +
+"    foreign\n" +
+"    suicide\n" +
+"    Darfur\n" +
+"    NGOs\n" +
+"    MSF\n" +
+"   ICRC\n" +
+"  U.N.\n" +
+"    injured");
+            modelAndView.addObject("response", gson.toJson(documents));
+            modelAndView.addObject("locations", gson.toJson(locations));
+            return modelAndView;
+        } catch (SolrServerException ex) {
+            Logger.getLogger(EndpointController.class.getName()).log(Level.SEVERE, null, ex);
+            return new ModelAndView(W6_VIEW);
+        }
+    }
     
+    @RequestMapping(value = "", method = RequestMethod.GET)
+    public ModelAndView map() throws IOException, SolrServerException
+    {
+        ArrayList<Article> documents = solrClient.getDocuments("*:*");
```

Comment:
Getting all documents? Only `response` filed is used, so query should set SolrFields through `setFields` method